Changelog.md

```diff
@@ -1,5 +1,27 @@
 ## Gazebo Utils 3.x
 
+## Gazebo Utils 3.1.0 (2024-11-15)
+
+1. **Baseline:** this includes all changes from 3.0.0 and earlier.
```

Comment:
I don't think we need this line for minor/patch releases, but not going to block on this.

Reply:
Removed.